editor/renames_map_3_to_4.cpp

```diff
@@ -644,7 +644,6 @@ const char *RenamesMap3To4::csharp_function_renames[][2] = {
 	{ "_SetPlaying", "SetPlaying" }, // AnimatedSprite3D
 	{ "_ToplevelRaiseSelf", "_TopLevelRaiseSelf" }, // CanvasItem
 	{ "_UpdateWrapAt", "_UpdateWrapAtColumn" }, // TextEdit
-	{ "AddAnimation", "AddAnimationLibrary" }, // AnimationPlayer
```

Comment:
As none of the commented out conversions are mentioned in the same way in the C# part I just removed it, can add it as a commented out part if desired